lib/SILOptimizer/Mandatory/TFUtilities.h

```diff
@@ -152,6 +160,19 @@ struct SILTensorOpInfo {
            operandClasses[operandNumber].second == OperandClass::InputElt;
   }
 
+  /// Returns the full name that this builtin would have if its operands
+  /// changed to the passed-in values.
+  std::string getBuiltinNameWithNewOperands(
+      const SmallVectorImpl<std::pair<StringRef, OperandClass>> &newOperandClasses) const {
+    std::string name = "__tfop_" + opName.str();
+    for (auto newOperandClass : newOperandClasses) {
```

Comment:
Use `const auto &newOperandClass` to prevent copies.

Reply:
done